modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/ExclusionAccessor.java

```diff
@@ -37,6 +38,9 @@
     @Query("select * from exclusion where base = ?")
     Result<Exclusion> getExclusions(UUID base);
```

Comment:
Need to keep this method as it is still being used in other places other than batch-assignment.